src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Venues/DeleteVenue.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core.DataManagement;
+using Dfc.CourseDirectory.Core.DataStore.Sql;
+using Dfc.CourseDirectory.Core.DataStore.Sql.Models;
+using Dfc.CourseDirectory.Core.DataStore.Sql.Queries;
+using Dfc.CourseDirectory.Core.Validation;
+using FluentValidation.Results;
+using MediatR;
+using OneOf;
+using OneOf.Types;
+
+namespace Dfc.CourseDirectory.WebV2.Features.DataManagement.Venues.Delete
```

Comment:
Similar to above, I think `DeleteRow` as a namespace (and file name) would be good here.

Reply:
done